src/main/java/core/basesyntax/dao/CsvDao.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.exception.CsvFileException;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class CsvDao implements FruitShopDao {
+
+    private static final int FIRST_LINE = 0;
+
+    @Override
+    public List<String> loadDataFromFile(Path readPath) {
+        try {
+            List<String> lines = Files.readAllLines(readPath);
+            if (!lines.isEmpty() && lines.get(FIRST_LINE).matches("\\D+,\\D+,\\D+")) {
+                lines.remove(FIRST_LINE);
```

Comment:
this custom line removal has nothing to do with the responsibility of reading the file, this class should just read/write the data it has received without any modifications. 
Because it'll be really confusing if you or you colleague were to use that class somewhere else, reading some data, facing the problem, and after several hours of debugging - you find out that it is removing the first line, and nobody really knows why 😄 because it just says `readDataFromFile()` and nothing about the skip of the line